### PR TITLE
Add function to remove card from page

### PR DESCRIPTION
**Pull request by JustinTorrance**

### CODE REVIEW

File: src/CharacterInfoCard.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ class CharacterInfoCard extends Component {
         </div>
         <button onClick={e => this.props.scrollCard(e)} className='left-button'>Left</button>
         <button onClick={e => this.props.scrollCard(e)} className='right-button'>Right</button>
-        <button className='delete-button'>Delete</button>
+        <button onClick={e => this.props.removeCard(e)} className='delete-button'>Delete</button>
```
Comment (jakelauer27) [inline src/CharacterInfoCard.js]: props down action up!

File: src/Characters.js [hunk 2]
```diff
@@ -67,6 +68,15 @@ class Characters extends Component {
     }
   }
 
+
+  removeCard(e) {
+    if (e.target.classList.contains('delete-button')) {
+      this.setState({
+        card: ''
```
Comment (jakelauer27) [inline src/Characters.js]: Sweet. Too easy

File: src/CharacterInfoCard.js (hunk 1 above)
Comment (libbyeh) [reply src/CharacterInfoCard.js]: Looks amazing, deletes as expected!

File: src/Characters.js (hunk 2 above)
Comment (libbyeh) [reply src/Characters.js]: agreed!

**Pull request merged**